### PR TITLE
Adds new blog post announcing opensearch hadoop

### PR DESCRIPTION
**Pull request by harshavamsi**
### Description
Adds a new blog post announcing the availability of the hadoop client
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by vagimeli — COMMENTED** [review]
See editorial review comments and changes. Please reach out with any questions.

File: _posts/2023-06-05-opensearch-hadoop-launch.markdown [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+meta_keywords: opensearch hadoop, apache spark, apache hive, apache hadoop, openseearch, mapreduce, hdfs
+meta_description: OpenSearch Hadoop is now generally available with support for multiple versions of OpenSearch to run on Spark and Hive.
+twittercard:
+  description: OpenSearch Hadoop is now generally available with support for multiple versions of OpenSearch to run on Spark and Hive.
+excerpt: We are excited to announce the release of the new OpenSearch-Hadoop connector. This tool enables efficient interaction between your Hadoop-based Big Data operations and OpenSearch clusters, supporting all versions of OpenSearch.
+---
+
+We are excited to announce the release of the new OpenSearch-Hadoop connector. This tool enables efficient interaction between your Hadoop-based Big Data operations and OpenSearch clusters, supporting all versions of OpenSearch.
+
+## OpenSearch-Hadoop connector features:
```
[the remark this answers is not in the document]
Comment (vagimeli) [reply _posts/2023-06-05-opensearch-hadoop-launch.markdown:19]: If you're referring to the connector `opensearch-hadoop` then we should write in code font.

**Review by nknize — CHANGES_REQUESTED** [review]
This is a good start. I like having the compatibility matrices. Might be good, though, to also add a simple "Getting Started Examples"?

Maybe an example on how to write to a dataframe in scala? 

e.g., 
```
val spark = SparkSession.builder().master("local[*]")
    .config("opensearch.nodes", "127.0.0.1").config("opensearch.net.http.auth.user", "admin").config("opensearch.net.http.auth.pass", "admin").config("opensearch.net.ssl", "true")
    .config("opensearch.batch.size.bytes", "1kb").config("opensearch.net.ssl.cert.allow.self.signed", "true")
    .getOrCreate()
```

or how to use it with pyspark like I demonstrate in my [comment on #153](https://github.com/opensearch-project/opensearch-hadoop/issues/153#issuecomment-1551501905).

I'm happy to add if you'd like.

File: _posts/2023-06-05-opensearch-hadoop-launch.markdown [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+| 1.0.1          | 3.2.4         | 2.12/2.13        |
+
+## Compatibility with AWS Glue
+
+| Client version | Spark version | Glue version(s) |
+| -------------- | ------------- | --------------- |
+| 1.0.1          | 2.4.4         | 2               |
+| 1.0.1          | 3.2.4         | 3/4             |
+
+This connector is a crucial tool for anyone looking to leverage the full power of OpenSearch alongside their Hadoop ecosystem. Download and start using it today.
```
Comment (vagimeli) [inline _posts/2023-06-05-opensearch-hadoop-launch.markdown:50]:
```suggestion
## Get started today!

This connector is a crucial tool for anyone looking to leverage the full power of OpenSearch alongside their Hadoop ecosystem. Download and start using it today.
```

File: _posts/2023-06-05-opensearch-hadoop-launch.markdown [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+meta_keywords: opensearch hadoop, apache spark, apache hive, apache hadoop, openseearch, mapreduce, hdfs
+meta_description: OpenSearch Hadoop is now generally available with support for multiple versions of OpenSearch to run on Spark and Hive.
+twittercard:
+  description: OpenSearch Hadoop is now generally available with support for multiple versions of OpenSearch to run on Spark and Hive.
+excerpt: We are excited to announce the release of the new OpenSearch Hadoop connector. This tool enables efficient interaction between your Hadoop-based Big Data operations and OpenSearch clusters, supporting all versions of OpenSearch.
+---
+
+We are excited to announce the release of the new OpenSearch-Hadoop connector. This tool enables efficient interaction between your Hadoop-based Big Data operations and OpenSearch clusters, supporting all versions of OpenSearch.
+
+## OpenSearch Hadoop connector features:
```
Comment (hdhalter) [inline _posts/2023-06-05-opensearch-hadoop-launch.markdown:19]: Can we add an opening paragraph here? For example, "The OpenSearch-Hadoop connector includes the following features:" (and remove the colon from the heading)

**Comment by harshavamsi** [comment]
> 

This would be awesome to have!

File: _posts/2023-06-05-opensearch-hadoop-launch.markdown (hunk 2 above)
[the remark this answers is not in the document]
Comment (harshavamsi) [reply _posts/2023-06-05-opensearch-hadoop-launch.markdown:50]: https://github.com/opensearch-project/opensearch-hadoop/releases -- we have the releases section here. 

The published connector is available https://central.sonatype.com/search?q=opensearch-hadoop and https://central.sonatype.com/search?q=opensearch-spark&smo=true

[the remark this answers is not in the document]
Comment (harshavamsi) [reply _posts/2023-06-05-opensearch-hadoop-launch.markdown:52]: @vagimeli the connector will work with 2.8 when it comes out, yes.

File: _posts/2023-06-05-opensearch-hadoop-launch.markdown [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+## Compatibility with AWS Glue
+
+| Client version | Spark version | Glue version(s) |
+| -------------- | ------------- | --------------- |
+| 1.0.1          | 2.4.4         | 2               |
+| 1.0.1          | 3.2.4         | 3/4             |
+
+## Get started today!
+
+The OpenSearch-Hadoop connector is a must-have tool for anyone looking to leverage the full power of OpenSearch alongside their Hadoop ecosystem. Download and get started today, available on the OpenSearch Downloads page.
```
Comment (vagimeli) [inline _posts/2023-06-05-opensearch-hadoop-launch.markdown:52]:
```suggestion
The OpenSearch-Hadoop connector is a must-have tool for anyone looking to leverage the full power of OpenSearch alongside their Hadoop ecosystem. Download and get started today, available on the OpenSearch [Download & Get Started] (https://opensearch.org/downloads.html) page.
```

File: _posts/2023-06-05-opensearch-hadoop-launch.markdown [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+authors:
+  - hvamsi
+  - nknize
+date: 2023-05-06 12:15:00 -0700
+categories:
+  - releases
+meta_keywords: opensearch hadoop, apache spark, apache hive, apache hadoop, openseearch, mapreduce, hdfs
+meta_description: OpenSearch Hadoop is now generally available with support for multiple versions of OpenSearch to run on Spark and Hive.
+twittercard:
+  description: OpenSearch Hadoop is now generally available with support for multiple versions of OpenSearch to run on Spark and Hive.
```
Comment (vagimeli) [inline _posts/2023-06-05-opensearch-hadoop-launch.markdown:13]:
```suggestion
  description: OpenSearch-Hadoop is now generally available with support for multiple versions of OpenSearch to run on Spark and Hive.
```

File: _posts/2023-06-05-opensearch-hadoop-launch.markdown [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+excerpt: We are excited to announce the release of the new OpenSearch Hadoop connector. This tool enables efficient interaction between your Hadoop-based Big Data operations and OpenSearch clusters, supporting all versions of OpenSearch.
```
Comment (vagimeli) [inline _posts/2023-06-05-opensearch-hadoop-launch.markdown:14]:
```suggestion
excerpt: We are excited to announce the release of the new OpenSearch-Hadoop connector. This tool enables efficient interaction between your Hadoop-based Big Data operations and OpenSearch clusters, supporting all versions of OpenSearch.
```

File: _posts/2023-06-05-opensearch-hadoop-launch.markdown [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+---
+
+We are excited to announce the release of the new OpenSearch-Hadoop connector. This tool enables efficient interaction between your Hadoop-based Big Data operations and OpenSearch clusters, supporting all versions of OpenSearch.
+
+## OpenSearch Hadoop connector features:
+
```
Comment (vagimeli) [inline _posts/2023-06-05-opensearch-hadoop-launch.markdown:20]: The following highlights the connector's features:

File: _posts/2023-06-05-opensearch-hadoop-launch.markdown (hunk 3 above)
Comment (wbeckler) [reply _posts/2023-06-05-opensearch-hadoop-launch.markdown]: > This is a good start. I like having the compatibility matrices. Might be good, though, to also add a simple "Getting Started Examples"?
> 
> Maybe an example on how to write to a dataframe in scala?
> 
> e.g.,
> 
> ```
> val spark = SparkSession.builder().master("local[*]")
>     .config("opensearch.nodes", "127.0.0.1").config("opensearch.net.http.auth.user", "admin").config("opensearch.net.http.auth.pass", "admin").config("opensearch.net.ssl", "true")
>     .config("opensearch.batch.size.bytes", "1kb").config("opensearch.net.ssl.cert.allow.self.signed", "true")
>     .getOrCreate()
> ```
> 
> or how to use it with pyspark like I demonstrate in my [comment on #153](https://github.com/opensearch-project/opensearch-hadoop/issues/153#issuecomment-1551501905).
> 
> I'm happy to add if you'd like.

Go for it!

**Comment by wbeckler** [comment]
> This is a good start. I like having the compatibility matrices. Might be good, though, to also add a simple "Getting Started Examples"?
> 
> Maybe an example on how to write to a dataframe in scala?
> 
> e.g.,
> 
> ```
> val spark = SparkSession.builder().master("local[*]")
>     .config("opensearch.nodes", "127.0.0.1").config("opensearch.net.http.auth.user", "admin").config("opensearch.net.http.auth.pass", "admin").config("opensearch.net.ssl", "true")
>     .config("opensearch.batch.size.bytes", "1kb").config("opensearch.net.ssl.cert.allow.self.signed", "true")
>     .getOrCreate()
> ```
> 
> or how to use it with pyspark like I demonstrate in my [comment on #153](https://github.com/opensearch-project/opensearch-hadoop/issues/153#issuecomment-1551501905).
> 
> I'm happy to add if you'd like.

Go for it!

**Comment by hdhalter** [comment]
Is anyone making any updates to this (@nknize )? We are targeting next week to publish it. Thanks!

**Comment by nknize** [comment]
> Is anyone making any updates to this (@nknize )? We are targeting next week to publish it. Thanks!

Yes. I'll put the example in tomorrow.

**Comment by wbeckler** [comment]
> > Is anyone making any updates to this (@nknize )? We are targeting next week to publish it. Thanks!
> 
> Yes. I'll put the example in tomorrow.

Hi Nick, this is still awaiting your input. Thank you!!

**Comment by pajuric** [comment]
@nknize @mnkugler @wbeckler - If you can make the final edits, update he blog date, and let @krisfreedain know when it's ready to go, we can get this posted to the blog tomorrow. Otherwise, we'll need to hold this until next Wednesday.

**Comment by nknize** [comment]
> Otherwise, we'll need to hold this until next Wednesday. 


Let's hold to Wednesday. I was working up the example with the published artifacts and noticed they don't support Spark 3. We may want to republish the Spark 3 artifacts before publishing the blog.

**Comment by pajuric** [comment]
@mnkugler and @wbeckler - Are we good to publish this today?

**Comment by wbeckler** [comment]
Still waiting on @nknize's changes.

**Comment by nknize** [comment]
@pajuric The blocker right now is that the released OpenSearched-Hadoop artifacts are [not compatible with Spark 3](https://central.sonatype.com/artifact/org.opensearch.client/opensearch-hadoop/1.0.1/dependencies). Thus the compatibility matrix in this blog post is not correct and the example code I'm providing will not work for the users / readers:

e.g., 

```
[error] Modules were resolved with conflicting cross-version suffixes in ProjectRef(uri("file:/...
[error]    org.apache.spark:spark-core _2.13, _2.11
```

```
ThisBuild / scalaVersion := "2.13.0"

lazy val root = (project in file("."))
  .settings(
    name := "opensearch-spark-example"
  )

libraryDependencies ++= Seq(
  "org.apache.spark" %% "spark-core" % "3.2.4" exclude("javax", "servlet") exclude("org.apache", "hadoop"),
  "org.opensearch.client" % "opensearch-hadoop" % "1.0.1",
  "org.antlr" % "antlr4-runtime" % "4.8",
  "org.codehaus.janino" % "commons-compiler" % "3.0.8",
  "org.codehaus.janino" % "janino" % "3.0.8"
)
```

We need to publish the Spark 3 compatible version which is built and packaged with the artifacts from the [spark/sql-30 module](https://github.com/opensearch-project/opensearch-hadoop/blob/main/spark/sql-30/build.gradle)

**Comment by nknize** [comment]
I opened an issue to move this forward: https://github.com/opensearch-project/opensearch-hadoop/issues/304

**Comment by pajuric** [comment]
@vagimeli @nknize - Just checking the status on this blog to see if there are any updates?

**Comment by vagimeli** [comment]
> @vagimeli @nknize - Just checking the status on this blog to see if there are any updates?

@pajuric I've not heard from the authors in a while. I'm adding them to this comment, as they need to provide the update. 

@nknize @harshavamsi Please update on the status of this blog. Is the text final and ready for an editorial review?

**Comment by pajuric** [comment]
@wbeckler @Xtansia - Please provide an update on the blog, as I understand it has been transferred over to you both.

**Comment by pajuric** [comment]
@wbeckler - Are we OK to close this blog?

**Pull request closed without merging**